dill/_dill.py

```diff
@@ -1829,7 +1595,7 @@ def save_wrapper_descriptor(pickler, obj):
 def save_cell(pickler, obj):
     try:
         f = obj.cell_contents
-    except:
+    except ValueError:
```

Comment:
Not sure if this is always a VaueError...

Reply:
I thought it would raise an `AttributeError` if there wasn't "cell contents", but it actually rose `ValueError` when the cell was empty. I've just found the documentation about this and it seems to be the case (see with `help(CellType)`):

```
Help on class cell in module builtins:

class cell(object)
 |  Create a new cell object.
 |  
 |   contents
 |     the contents of the cell. If not specified, the cell will be empty,
 |     and 
 |  further attempts to access its cell_contents attribute will
 |     raise a ValueError.
(...)
```